vision/genesis_principles.md

```diff
@@ -0,0 +1,14 @@
+# ₿trust Genesis Principles
+₿trust’s Genesis Principles are an immutable set of principles governing its operation and outlining its vision. They exist to guide the board and representatives of the trust, and to act as an accountability safeguard against bad actors.  
+
+Board members will take ownership of this vision, and act in good faith to protect the interests and integrity of the organization. The specific implementation of these principles is the prerogative of the board and its selected representatives.
+
+1. Bitcoin will provide a monetary system that enhances financial sovereignty for all. ₿trust will support Bitcoin exclusively. In the event of a hard fork it will follow the chain endorsed by the user majority.
```

Comment:
```suggestion
1. Bitcoin will provide a monetary system that enhances financial sovereignty for all. ₿trust will support Bitcoin exclusively. In the event of a hard fork, it will follow the chain endorsed by the user majority.
```

Reply:
Hm, do we need a comma here? My grammar is 0/10, but it _feels_ like it breaks the sentence strangely.

Reply:
Yes, I believe the comma is needed here as "_In the event of a hard fork_" reads more like an introductory phrase.